pkg/cmd/context/create/create.go

```diff
@@ -0,0 +1,109 @@
+package create
+
+import (
+	"context"
+
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/context/contextcmdutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/icon"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/iostreams"
+	"github.com/redhat-developer/app-services-cli/pkg/core/localize"
+	"github.com/redhat-developer/app-services-cli/pkg/core/logging"
+	"github.com/redhat-developer/app-services-cli/pkg/core/servicecontext"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/profileutil"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	IO             *iostreams.IOStreams
+	Logger         logging.Logger
+	Connection     factory.ConnectionFunc
+	localizer      localize.Localizer
+	Context        context.Context
+	ServiceContext servicecontext.IContext
+
+	name string
+}
+
+// NewCreateCommand creates a new command to create contexts
+func NewCreateCommand(f *factory.Factory) *cobra.Command {
+
+	opts := &options{
+		Connection:     f.Connection,
+		IO:             f.IOStreams,
+		Logger:         f.Logger,
+		localizer:      f.Localizer,
+		ServiceContext: f.ServiceContext,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "create",
```

Comment:
That has been removed after going through how kubectl does it. Since we might need to create multiple contexts for same project, we shouldn't set auto use. Will having a `--use` flag with default `false` be useful?

Reply:
IMHO Using created context by default skips extra steps and it is logical for many users. Then flag might not be needed

Reply:
I don't think we need any `use` flag at all.